src/index.js

```diff
@@ -31,5 +32,5 @@ const render = (store) => {
     </Provider>, root);
 };
 
-export const store = compose(applyMiddleware(thunk))(createStore)(reducer);
+const store = createStore(reducer, applyMiddleware(thunk));
```

Comment:
TBH: i would leave `compose` as it was before - it's easier to add new middlewares in future.